### DISCUSSION TITLE
nfdump -C -> config file syntax

### DISCUSSION THREAD

**Discussion by jniggemann** [phaag/nfdump#471] [General]
Using 1.7.1, I see `nfdump` has an option `-C <file>` "read optional config file". I'd like to set the data directory in there and tried with `cache_dir=/var/cache/nfdump`, but get `Failed to parse config file nfdump.conf: line 1: extra chars after value`.
What options can I set there and what's the correct syntax?

**Comment by phaag** [comment]
The optional config file does not include all options, which are available on the command line.  The list file `nfdump.conf.dist` contains all available options as comments with their default value.
